### PR TITLE
Operator + Serving web console intall docs for Serverless

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/serverless-install-web-console.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+= Installing the {ServerlessOperatorName}
+
+This procedure describes how to install and subscribe to the {ServerlessOperatorName} from the OperatorHub using the {product-title} web console.
+
+.Procedure
+
+. Navigate in the web console to the *Catalog* → *OperatorHub* page.
+. Scroll, or type they keyword *Serverless* into the *Filter by keyword box* to find the {ServerlessOperatorName}.
++
+image::serverless-search.png[{ServerlessOperatorName} in the {product-title} web console]
```
Comment (mgencur) [inline modules/serverless-install-web-console.adoc:15]: This image shows a tile with an "already installed" Serverless (see the tick). Not sure if this is a big problem but it's not exactly what the user will see.

File: modules/serverless-install-web-console.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+. Read the information about the Operator and click *Install*.
++
+image::serverless-operator.png[{ServerlessOperatorName} information]
```
Comment (mgencur) [inline modules/serverless-install-web-console.adoc:19]: This page shows version 1.5 and it shows "This is a Tech Preview Release!" . Hmm, we should probably find a new screenshot.

File: modules/serverless-install-web-console.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+.. The *Installation Mode* is *All namespaces on the cluster (default)*. This mode  installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.
+.. The *Installed Namespace* will be `openshift-operators`.
+.. Select an *Update Channel*.
+... The *4.3* channel will enable installation of the latest stable release of the {ServerlessOperatorName}.
+... The *preview-4.3* channel enables installation of the latest preview version of the {ServerlessOperatorName}, which may contain features that are not yet available from the *4.3* update channel.
+.. Select *Automatic* or *Manual* approval strategy.
+. Click *Subscribe* to make the Operator available to the selected namespaces on this {product-title} cluster.
+. From the *Catalog* → *Operator Management* page, you can monitor the {ServerlessOperatorName} Subscription’s installation and upgrade progress.
+.. If you selected a *Manual* approval strategy, the Subscription’s upgrade status will remain *Upgrading* until you review and approve its Install Plan. After approving on the *Install Plan* page, the Subscription upgrade status moves to *Up to date*.
```
Comment (mgencur) [inline modules/serverless-install-web-console.adoc:33]:
```suggestion
.. If you selected a *Manual* approval strategy, the Subscription’s upgrade status will remain *Upgrading* until you review and approve its *Install Plan*. After approving on the *Install Plan* page, the Subscription upgrade status moves to *Up to date*.
```

File: modules/serverless-install-serving-yaml.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+. Create a file named `serving.yaml`.
+. Copy the following sample YAML into `serving.yaml`:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+ name: knative-serving
```
Comment (mgencur) [inline modules/serverless-install-serving-yaml.adoc:18]:
```suggestion
  name: knative-serving
```

File: modules/serverless-install-serving-yaml.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+ namespace: knative-serving
```
Comment (mgencur) [inline modules/serverless-install-serving-yaml.adoc:19]:
```suggestion
  namespace: knative-serving
```

**Review by mgencur — CHANGES_REQUESTED** [review]
Hi @abrennan89 , this look pretty good! The text is nice. The images will need to be replaced with up-to-date ones though.

File: modules/serverless-install-web-console.adoc (hunk 3 above)
Comment (abrennan89) [reply modules/serverless-install-web-console.adoc:33]: I don't think this really needs that kind of emphasis because it's referring this time to an object as opposed to the page name?
e.g. Subscription doesn't get *strong* text either :) actually subscription and install plan should probably be lower case in these instances.

@jboxman care to pitch in? I know you were talking about capitalization of terms etc recently in a call :slightly_smiling_face:

**Comment by mgencur** [comment]
QE Approved

File: modules/serverless-install-serving-web-console.adoc [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
++
+image::serverless-installed-operator.png[Installed Operators page]
+. Click the *Create Knative Serving* button.
++
+image::serverless-create-serving.png[Knative Serving tab]
+. In the *Create Knative Serving* page, you can choose to configure the `KnativeServing` object by using either the default form provided, or by editing the YAML.
++
+* Using the form is recommended for simpler configurations that do not require full control of `KnativeServing` object creation.
++
+If you are configuring the `KnativeServing` object using the form, make any changes that you want to implement for your Knative Serving deployment (optional), then click *Create*.
```
Comment (lamek) [inline modules/serverless-install-serving-web-console.adoc:21]: Minor - the word "(optional)" threw me off here. I suggest moving it to the beginning of the step, and breaking "Click Create." into a new step.

File: modules/serverless-install-serving-web-console.adoc [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
++
+image::serving-form-view.png[Create Knative Serving in Form View]
++
+* Editing the YAML is recommended for more complex configurations that require full control of `KnativeServing` object creation. You can access the YAML by clicking the *edit YAML* link in the top right of the *Create Knative Serving* page.
++
+If you are configuring the `KnativeServing` object by editing the YAML, make any changes to the YAML that you want to implement for your Knative Serving deployment (optional), then click *Create*.
```
Comment (lamek) [inline modules/serverless-install-serving-web-console.adoc:27]: Please see my note on "(optional)" above.

File: modules/serverless-install-web-console.adoc [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+// * /serverless/installing_serverless/installing-openshift-serverless.adoc
+
+[id="serverless-install-web-console_{context}"]
+= Installing the {ServerlessOperatorName}
+
+This procedure describes how to install and subscribe to the {ServerlessOperatorName} from the OperatorHub using the {product-title} web console.
+
+.Procedure
+
+. Navigate in the web console to the *Catalog* → *OperatorHub* page.
```
Comment (lamek) [inline modules/serverless-install-web-console.adoc:12]: Reword this to keep consistency with a similar step above:
In the {product-title} web console, navigate to the *Catalog* → *OperatorHub* page.
https://github.com/openshift/openshift-docs/pull/21074/files#diff-aa00c6cbca126bf4d6ddf98f09f49e5aR25

File: modules/serverless-install-serving-web-console.adoc [hunk 9]
```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+//  * serverless/installing_serverless/installing-knative-serving.adoc
+
+[id="serverless-install-serving-web-console_{context}"]
+= Installing Knative Serving using the web console
+
+.Procedure
+. Navigate in the web console to the *Catalog* → *OperatorHub* → *Installed Operators* page.
```
Comment (lamek) [inline modules/serverless-install-serving-web-console.adoc:9]: Reword this to keep consistency with a similar step above:
In the {product-title} web console, navigate to the *Catalog* → *OperatorHub* → *Installed Operators* page.
https://github.com/openshift/openshift-docs/pull/21074/files#diff-aa00c6cbca126bf4d6ddf98f09f49e5aR25

File: modules/serverless-install-web-console.adoc [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+. Scroll, or type they keyword *Serverless* into the *Filter by keyword box* to find the {ServerlessOperatorName}.
++
+image::serverless-search.png[{ServerlessOperatorName} in the {product-title} web console]
+
+. Read the information about the Operator and click *Install*.
```
Comment (lamek) [inline modules/serverless-install-web-console.adoc:17]: NIT: I'd change "**Read** the information..." to "**Review** the information..."

File: modules/serverless-install-web-console.adoc [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
++
+image::serverless-operator.png[{ServerlessOperatorName} information]
+
+. On the *Create Operator Subscription* page:
++
+image::serverless-create-sub.png[Create Operator Subscription page]
+
+.. The *Installation Mode* is *All namespaces on the cluster (default)*. This mode  installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.
```
Comment (lamek) [inline modules/serverless-install-web-console.adoc:25]: NIT: extra space after "This mode..."

**Review by lamek — COMMENTED** [review]
Few suggestions and a couple NITS. Otherwise LGTM. Thanks.
